llm_bioinformatics_research/src/SignupScreen/SignupScreen.js

```diff
@@ -103,182 +115,193 @@ function SignUpScreen() {
+          <Button
+            id="signup-button"
+            type="submit"
+            variant="contained"
+            color="primary"
+            fullWidth
+            className="signup-button"
             data-testid="signup-button"
           >
             Sign Up
           </Button>
           {signupSuccess && (
-            <Typography 
-              variant="body1" 
-              color="success.main" 
-              sx={{ mt: 2 }}
+            <Typography
+              variant="body1"
+              color="success.main"
+              className="signup-success"
               data-testid="signup-success-message"
             >
               Successfully signed up! You will be redirected to the login page in 5 seconds.
             </Typography>
           )}
         </form>
-        <Box sx={{ mt: 2, textAlign: 'center' }}>
-            <Button
-              variant="outlined"
-              fullWidth
-              startIcon={<GoogleIcon />}
-              sx={{
-                mb: 2,
-                borderColor: '#db4437',
-                color: '#db4437',
-                borderRadius: '50px',
-                padding: '10px 20px',
-                textTransform: 'none',
-              }}
-            >
-              Sign in with Google
-            </Button>
-
-            <Button
-              variant="outlined"
-              fullWidth
-              startIcon={<GitHubIcon />}
-              sx={{
-                borderColor: '#000',
-                color: '#000',
-                borderRadius: '50px',
-                padding: '10px 20px',
-                textTransform: 'none',
-              }}
-            >
-              Sign in with GitHub
-            </Button>
-          </Box>
-
-          <Box sx={{ mt: 2, textAlign: 'center' }}>
-            <Link component={RouterLink} to="/login" variant="body2">
-              Already have an account? Login
-            </Link>
-          </Box>
+        <Box className="social-signup">
+          <Button
+            variant="outlined"
+            fullWidth
+            startIcon={<GoogleIcon />}
+            sx={{
+              mb: 2,
+              borderColor: '#db4437',
+              color: '#db4437',
+              borderRadius: '50px',
+              padding: '10px 20px',
+              textTransform: 'none',
+              '&:hover': {
+                backgroundColor: '#db4437',
+                color: '#fff',
+              },
+            }}
+          >
+            Sign in with Google
+          </Button>
+  
+          <Button
+            variant="outlined"
+            fullWidth
+            startIcon={<GitHubIcon />}
+            sx={{
+              borderColor: '#000',
+              color: '#000',
+              borderRadius: '50px',
+              padding: '10px 20px',
+              textTransform: 'none',
+              '&:hover': {
+                backgroundColor: '#000',
+                color: '#fff',
+              },
+            }}
```

Comment:
I see that you have created a SignupScreen.css file to separate out stylistic components. However, the SignupScreen.js file still contains css code that needs to be extracted out. Before you merge your code, make sure the .js file is clean of stylistic components.